deploy_storybook_to_s3.sh

```diff
@@ -19,6 +20,20 @@ echo "I am building a storybook"
 yarn storybook:build
 
 echo "Deploying storybook to $BUCKET as $BRANCH"
-s3cmd put -r -P public/storybook/* s3://$BUCKET/$BRANCH/
+s3cmd put -r -P $LOCAL_DIRECTORY/* s3://$BUCKET/$BRANCH/
+
+echo "Setting Content-Type to application/javascript for all .js files"
+find "$LOCAL_DIRECTORY" -type f -name "*.js" | while read FILE; do
+    RELATIVE_PATH="${FILE#"$LOCAL_DIRECTORY/"}"
+    echo "s3://$BUCKET/$BRANCH/$RELATIVE_PATH"
+    s3cmd modify --add-header="Content-Type: application/javascript" s3://$BUCKET/$BRANCH/$RELATIVE_PATH
+done
+
+echo "Setting Content-Type to text/css for all .css files"
+find "$LOCAL_DIRECTORY" -type f -name "*.css" | while read FILE; do
+    RELATIVE_PATH="${FILE#"$LOCAL_DIRECTORY/"}"
+    echo "s3://$BUCKET/$BRANCH/$RELATIVE_PATH"
+    s3cmd modify --add-header="Content-Type: text/css" s3://$BUCKET/$BRANCH/$RELATIVE_PATH
+done
```

Comment:
There are so many assets and it takes so long... I hope we find a better solution soon.